src/sass/card/card.scss

```diff
@@ -0,0 +1,135 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+}
+
+.card__container {
+    position: relative;
+    height: $card-size-default;
+}
+
+@media screen and (max-width: $_screen-size-MD) {
+    .card {
+        width: $card-size-large;
+
+        .card__container {
+            height: $card-size-large;
+        }
+    }
+}
+
+.card__content {
+    display: flex;
+    height: inherit;
+    width: inherit;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    background-color: var(--color-background-secondary, #f7f7f7);
+    border-radius: var(--border-radius-100, 16px);
+    padding: var(--spacing-150, 12px);
+}
+
+.card__content .progress-bar {
+    width: 100%;
+}
+
+.card__content--image {
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.card__content--overflow {
+    opacity: 0.5;
+}
+
+.card .card__action {
+    @include bubble-close();
+
+    position: absolute;
+    top: var(--spacing-100, 8px);
+    right: var(--spacing-100, 8px);
+    margin: 0;
+    background: var(--color-background-on-secondary, #fff);
```

Comment:
Use the mixin for the background please.
Make sure the first variable is not defined and relative to the component.
https://github.com/eBay/skin/blob/18.3.0/src/sass/lightbox-dialog/lightbox-dialog.scss#L177
So maybe @include background-color-token(card-action-color, color-background-on-secondary);

Reply:
I'm not following, why would we provide a variable that is not defined in the first arg for the mixin?